app/src/main/java/com/example/life_is_well/HealthMacroTracker.kt

```diff
@@ -0,0 +1,58 @@
+package com.example.life_is_well
+
+import android.app.DatePickerDialog
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.TextView
+import android.widget.Toast
+import java.text.SimpleDateFormat
+import java.util.*
+
+class PhysicalMacroTracker : AppCompatActivity() {
+
+    private var tvSelectedDate : TextView? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_health_macro_tracker)
+
+        val btnDateClicker : Button = findViewById(R.id.btnDateClicker)
+        tvSelectedDate = findViewById(R.id.tvSelectedDate)
+
+        btnDateClicker.setOnClickListener {
+
+            clickDateClicker()
+        }
+    }
+
+    private fun clickDateClicker(){
+
+        val myCalendar = Calendar.getInstance()
+        val year = myCalendar.get(Calendar.YEAR)
+        val month = myCalendar.get(Calendar.MONTH)
+        val day = myCalendar.get(Calendar.DAY_OF_MONTH)
+        val dpd = DatePickerDialog(this, DatePickerDialog.OnDateSetListener { view, selectedYear, selectedMonth, selectedDayOfMonth ->
+
+            Toast.makeText(this, "Year was $selectedYear, Month was ${selectedMonth + 1}, Day is $selectedDayOfMonth" , Toast.LENGTH_LONG).show()
+
+            val selectedDate = "${selectedMonth + 1}/$selectedDayOfMonth/$selectedYear"
```

Comment:
Curious about what the `selectedMonth-1`, `theDate.time/60000`, and the `System.currentTimeMillis() - 86400000` operations are for? Curious to see if it uses UTC or not.

Reply:
Noting this does not have to block the merge, but is something I wanted to ask about.